### PR TITLE
docs: use long options & fix conda install tex stack command

### PR DESCRIPTION
**Pull request by justin-f-perez**
* Use long options for conda install commands
* Fix conda install command for tex stack (missing `--channel`)

## References

I checked issue tracker; none open for this. Seems too minor but I'll open an issue if needed. I attempted to setup an environment to build the docs but it failed (and I'll open an issue for that soon as I have time- unfortunately I have some hard deadlines coming up that need attention and already spent too much time on this 😮‍💨 )

## Code changes

Mainly just makes conda install commands a bit more readable for the uninitiated and fixed the command for the tex stack by adding `--channel`

## User-facing changes

Fixes conda install command for tex stack

## Chores

- [ ] linted 
- [ ] tested 
- [ ] documented 
- [ ] changelog entry

### CODE REVIEW

File: docs/Language Servers.ipynb [hunk 1]
```diff
@@ -225,7 +225,7 @@
     "servers; you can get it with:\n",
     "\n",
     "```bash\n",
-    "conda install -c conda-forge nodejs\n",
+    "conda install --yes --channel conda-forge nodejs\n",
```
Comment (krassowski) [inline docs/Language Servers.ipynb:228]:
```suggestion
    "conda install --channel conda-forge nodejs\n",
```

Let's not add `--yes` - conda resolution can lead to drastic environment changes, I would prefer to let the user decide if they want to proceed.

File: docs/Language Servers.ipynb [hunk 2]
```diff
@@ -318,7 +318,7 @@
     "#### Example: Getting a $\\LaTeX$ stack\n",
     "\n",
     "```bash\n",
-    "conda install -y conda-forge tectonic texlab chktex\n",
+    "conda install --yes --channel conda-forge tectonic texlab chktex\n",
```
Comment (krassowski) [inline docs/Language Servers.ipynb:321]:
```suggestion
    "conda install --channel conda-forge tectonic texlab chktex\n",
```

Let's remove `-y`, it looks like it was a typo.

**Review by krassowski — APPROVED** [review]
Thank you!

File: docs/Language Servers.ipynb (hunk 1 above)
Comment (justin-f-perez) [reply docs/Language Servers.ipynb]: This is an excellent point, thanks for the feedback!

**Pull request merged**